AcademyResidentInformationApi/V1/Gateways/AcademyGateway.cs

```diff
@@ -46,9 +50,9 @@ where string.IsNullOrEmpty(lastname) || person.LastName.ToLower().Replace(" ", "
                 }
                 ).Skip(cursor).Take(limit).ToList().ToDomain();
         }
-        private static string StripString(string str)
+        private static string GetSearchPattern(string str)
         {
-            return str?.ToLower().Replace(" ", "");
+            return $"%{str?.ToLower().Replace(" ", "")}%";
```

Comment:
Do we still need to call `.ToLower()` if the result will be used in an `ILIKE` statement?

Reply:
Good point, No we don't